### PR TITLE
Merge branch 'release-22.1.0' into 'release-22.1'

### PR DESCRIPTION
**Pull request by celiala**
This commit merges `release-22.1.0` into `release-22.1`. 

This is a noop, as all backports in `release-22.1.0` have a corresponding backport in `release-22.1`. Full list of backports accounted for by the merge:

- kv/bulk: pull ingest stats into own struct
- kv/bulk: track send-wait by store
- sql: use BulkNormalPri for index backfiller transactions
- sql: enforce admin role for resetting sql stats and index usage stats
- kv/bulk: remove initial batcher split
- kv/bulk: parallelize sending SSTs due to range bounds
- ci: fix optimizer_tests_impl.sh
- sql: rewrite sql stats compaction job to avoid scanning mvcc garbage
- kvserver: change default for kv.raft_log.loosely_coupled_truncation.enabled to false
- roachtest: configure ORM tests to use SCRAM
- sql: fix "no user specified" errors in statement bundles
- kv/bulk: pull lastRange out of mu
- roachtest: fix django nightly test
- cli: persist new replica ID in `unsafe-remove-dead-replicas`
- kv/bulk: ensure batcher always has a non-nil monitor
- roachtest: fix sqlalchemy to install pip
- ui: update width of timescale component
- row: mark sql.distsql.use_streamer.enabled as TenantReadOnly setting
- bazel: refactor logic out of `pkg/cmd/mirror`
- bazel: provide method for constructing `distdir` for offline builds
- docker: fix docker image to suit the memory storage
- storageccl: Don't disable data key rotation after restart
- kvserver: version gate locality-aware load-based rebalancing
- roachtest: add mixed-version variants of the `rebalance/by-load` roachtests
- bazel: run `returncheck` lint in `nogo`
- lint: un-skip `CODEOWNERS` check for bazel
- git: remove `distdir_files.bzl` from `git` text diffs
- kvtenantccl: fix tenant upgrade test on new versions
- release: use preflight tool for RedHat images
- kvserver/loqrecovery: persist new replica ID in `RaftReplicaID`
- v22.1.0
- backupccl: allow secondary tenants to write pts records during backup
- colexec: fix sort chunks with disk spilling in very rare circumstances
- sql,migration: ensure cluster version never regresses
- licenses: updating licenses for 22.1
- clusterversion: mint 22.1 cluster version
- util/tracing: fix crash in StreamClientInterceptor
- acceptance: skip `TestComposeGSSPython`

### CODE REVIEW

File: pkg/acceptance/compose_test.go [hunk 1]
```diff
@@ -31,6 +32,7 @@ func TestComposeGSS(t *testing.T) {
 }
 
 func TestComposeGSSPython(t *testing.T) {
+	skip.WithIssue(t, 81254)
```
Comment (otan) [inline pkg/acceptance/compose_test.go:35]: we want to remove this backport (revert a419dff5aebbc62e34cbd47bd898d3e4492f4705)

Comment (celiala) [reply pkg/acceptance/compose_test.go:35]: Thanks! Ack, I'll revert this.

It seems that `git merge` is having trouble understanding that a commit in both branches was then superseded by a subsequent commit in just the release-22.1 branch. 

In doing an audit, it looks like all release-22.1.0 backports have a corresponding release-22.1 backports, except for this:

* https://github.com/cockroachdb/cockroach/pull/80574

I'll update this merge commit accordingly.

**Comment by celiala** [comment]
> Let's enumerate the changes we want to merge to 22.1 and ask the original authors if they should be merged. Maybe they would prefer backporting those and linking to the original issue instead. What do you think?

Agree that a noop would be ideal; good idea to loop in original authors. 

Hi @aayushshah15 - it looks like https://github.com/cockroachdb/cockroach/pull/80574 was backported to the temporary `release-22.1.0` branch (for `v22.1.0`), but there's no equivalent backport for the `release-22.1` branch (for `v22.1.1+`). Let us know if this should be merged into the `release-22.1` branch, or if you would prefer a separate backport, to link to the original issue instead? 

Thanks!

**Comment by celiala** [comment]
Re https://github.com/cockroachdb/cockroach/pull/80574 - the team is merging a separate backport to release-22.1 (https://github.com/cockroachdb/cockroach/pull/82168). Once #82168 is merged, a rebase of this PR should result in a noop.

**Comment by celiala** [comment]
Hi @rail this is RFAL - this has been rebased on top of latest release-22.1 branch, which includes the backported https://github.com/cockroachdb/cockroach/pull/82168 - result is a noop.

**Review by rail — APPROVED** [review]
Wheeeee. 0 files changed.

**Comment by celiala** [comment]
TFTR!

`pkg/kv/kvserver/rangefeed/rangefeed_test_/rangefeed_test` looks like a flake

**Pull request merged**